drizzle-zod/src/mysql/index.ts

```diff
@@ -0,0 +1,243 @@
+import { GetColumnConfig } from 'drizzle-orm';
+import {
+	AnyPgColumn,
+	AnyPgTable,
+	GetTableConfig,
+	PgBigInt53,
+	PgBigInt64,
+	PgBigSerial53,
+	PgBigSerial64,
+	PgBoolean,
+	PgDate,
+	PgDoublePrecision,
+	PgEnumColumn,
+	PgEnumColumnConfig,
+	PgInteger,
+	PgInterval,
+	PgNumeric,
+	PgReal,
+	PgSerial,
+	PgSmallInt,
+	PgSmallSerial,
+	PgText,
+	PgTextConfig,
+	PgTime,
+	PgTimestamp,
+	PgUUID,
+	PgVarchar,
+} from 'drizzle-orm/mysql-core';
```

Comment:
Not sure, how Pg columns were imported from `mysql-core` module

Reply:
I just used the GitHub code editor to copy and paste and make a single change to demo a quick way to get a MySQL version. I wanted feedback before investing too much time, and also wasn't sure if drizzle was open for PRs.

Reply:
Yeah, drizzle is open for PRs

Reply:
I will also take on your comments and attempt to resolve them when I can. Thanks for the review!